callback.go

```diff
@@ -353,13 +353,27 @@ func callbackRetNil(ctx *C.sqlite3_context, v reflect.Value) error {
 	return nil
 }
 
+func callbackRetAny(ctx *C.sqlite3_context, v reflect.Value) error {
```

Comment:
Probably should be `callbackRetGeneric` for consistency with `callbackArgGeneric`. Or rename the other one to `callbackArgAny`.

Reply:
Adjusted, thanks!